install_config/topics/vsphere_prerequisites.adoc

```diff
@@ -3,128 +3,136 @@
+    export GOVC_PASSWORD='vCenter Password'
+    export GOVC_INSECURE=1
+----
 
-. Create link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-031BDB12-D3B2-4E2D-80E6-604F304B4D0C.html[a
-VM folder] and move {product-title} Node VMs to this folder.
+=== Permissions
 
-. Verify that the Node VM names complies with the regex `[a-z](([-0-9a-z]+)?[0-9a-z])?(\.[a-z0-9](([-0-9a-z]+)?[0-9a-z])?)*`.
-+
-[IMPORTANT]
-====
-VM Names cannot:
+Create and assign roles to the vSphere Cloud Provider. A vCenter user with the required set of link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html[privileges] is required. 
 
-* Begin with numbers.
-* Have any capital letters.
-* Have any special characters except `-`.
-* Be shorter than three characters and longer than 63 characters.
-====
+In general, the vSphere user designated to the vSphere Cloud Provider should have the following permissions:
 
-. Set the `disk.EnableUUID` parameter to `true` for each Node VM. This ensures that the VMware vSphere's Virtual Machine Disk (VMDK) always presents a consistent UUID to the VM, allowing the disk to be mounted properly.
-+
-For every vSphere virtual machine node that will be participating in the cluster, follow the steps below using the vSphere console:
-+
-. Navigate to *VM properties* -> *VM Options* -> *Advanced* -> *Configuration* *Parameters* -> *disk.enableUUID=TRUE*
-+
-.. Set up the GOVC environment:
-+
-[source,bash]
-----
-curl -LO https://github.com/vmware/govmomi/releases/download/v0.15.0/govc_linux_amd64.gz
-gunzip govc_linux_amd64.gz
-chmod +x govc_linux_amd64
-cp govc_linux_amd64 /usr/bin/govc
-export GOVC_URL='vCenter IP OR FQDN'
-export GOVC_USERNAME='vCenter User'
-export GOVC_PASSWORD='vCenter Password'
-export GOVC_INSECURE=1
-----
+* `Read` permission on the parent entities of the node VMs such as `folder`, `host`, datacenter, datastore folder, datastore cluster, etc.
+* `VirtualMachine.Inventory.Create/Delete` permission on the vsphere.conf defined resource pool - this is used to create/delete dummy VMs.
 
-.. Find the Node VM paths:
-+
-[source,bash]
-----
-govc ls /datacenter/vm/<vm-folder-name>
-----
+See the link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.security.doc/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html[vSphere
+Documentation Center] for steps to create a custom role, user, and role
+assignment.
 
-.. Set `disk.EnableUUID` to `true` for all VMs:
-+
-[source,bash]
-----
-govc vm.change -e="disk.enableUUID=1" -vm='VM Path'
-----
+vSphere Cloud Provider supports OpenShift clusters spanning across multiple vCenters. Make sure that all above privileges are correctly set for all vCenters.
 
-[NOTE]
-====
-If {product-title} node VMs are created from a template VM, then
-`disk.EnableUUID=1` can be set on the template VM. VMs cloned from this
-template inherit this property.
-====
+==== Dynamic Provisioning Permissions
```

Comment:
Procedure modules can't have additional subheadings.

Reply:
Removed the formatting.